packages/astro/src/core/util.ts

```diff
@@ -18,9 +18,11 @@ export function isURL(value: unknown): value is URL {
 }
 /** Check if a file is a markdown file based on its extension */
 export function isMarkdownFile(fileId: string, option?: { suffix?: string }): boolean {
+	// Strip query string
+	const id = fileId.split("?")[0];
```

Comment:
Don't we have a stripQueryParams util somewhere?

Reply:
I couldn't find one. There's `getFileInfo` which does return an id with the params stripped - and which the markdown plugin calls immediately after, but it does lots of other stuff which is probably overkill for something that will be called on every module. Better to have something extra simple when it's in the kinda-hot path.

Reply:
We do! :) `removeQueryString`

Reply:
Oh!